.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+sudo: false
+
+services:
+  - docker
+
+env:
+  global:
+
+install:
+  - export GIT_FULL_HASH=`git rev-parse HEAD`
+  - bash .keep_alive.sh &  # Generate output periodically.
+  - pip install jupyter-repo2docker
+  - pip list
+
+script:
+  - repo2docker . --no-run
```

Comment:
Oops, I led you astray here. I think it has to be

```
repo2docker --no-run .
```

Reply:
@danielballan  Let's see if it helps